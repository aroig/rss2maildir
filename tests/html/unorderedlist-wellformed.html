<p>This is a well formed unordered list.</p>
<ul>
    <li>The first item</li>
    <li>The second item</li>
    <li>The third item</li>
</ul>
