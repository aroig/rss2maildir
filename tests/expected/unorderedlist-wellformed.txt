This is a well formed unordered list.

 * The first item
 * The second item
 * The third item
